### DISCUSSION TITLE
How to see large numbers with dot (.) separation (123.456.789)

### DISCUSSION THREAD

**Discussion by Kakoks** [Ulauncher/Ulauncher#1210] [Q&A] [answered]
So, i have been using ulauncher since i can't build albert and overall it is a very good launcher and i really like it, but i do all day on work a bunch of small multiplications and additions, and numbers just show like this 20*1000=20000 instead of 20*1.000=20.000. I have learnt to read like this for some time but its getting a bit annoying, is there anything that can be made here by me? or is this something that needs to be changed for the next release?

**Comment by friday** [comment, marked as answer]
See #937 #869 https://github.com/Ulauncher/Ulauncher#%EF%B8%8F-this-is-the-branch-for-the-unreleased-v6-release-if-you-want-the-v5-branch-version-go-here

**Reply by Kakoks** [reply]
So as far as i'm understanding, is gonna come with v6
